.github/workflows/build-wheel-linux-x86_64.yaml

```
@@ -376,14 +382,9 @@ jobs:
         name: catalyst-manylinux2014_x86_64-wheel-py-${{ matrix.python_version }}.zip
         path: dist
 
-    - name: Set up Python ${{ matrix.python_version }}
-      uses: actions/setup-python@v4
-      with:
-        python-version: ${{ matrix.python_version }}
-
     - name: Install Python dependencies
       run: |
-        python${{ matrix.python_version }} -m pip install pytest pytest-xdist
+        python${{ matrix.python_version }} -m pip install pytest pytest-xdist scipy
```

I'm hesitant whether this is needed here (and in fact may be detrimental). At this point we are testing the built package, any (mandatory) dependencies should be handled by the package itself, and we should only install testing requirements or optional feature dependencies here.

```suggestion
python${{ matrix.python_version }} -m pip install pytest pytest-xdist
```